containerserver/sqlite_backend.go

```diff
@@ -100,6 +101,20 @@ type sqliteContainer struct {
 
 var _ Container = &sqliteContainer{}
 
+func (db *sqliteContainer) quarantine() error {
```

Comment:
can we combine these 2 quarantine functions somewhere and have it take db.containerFile as an arg?